controller/smartlight_controller.go

```diff
@@ -50,37 +50,37 @@ func GetSmartLights(c *gin.Context) {
 		value := iterator.Value() // Value of the current row
 		obj := gin.H{
 			"fields": gin.H{
-					"data_counter_0d_0": value["data_counter_0d_0"],
-					"data_counter_0d_1": value["data_counter_0d_1"],
-					"fCnt": value["fCnt"],
-					"rxInfo_altitude_0": value["rxInfo_altitude_0"],
-					"rxInfo_latitude_0": value["rxInfo_latitude_0"],
-					"rxInfo_loRaSNR_0": value["rxInfo_loRaSNR_0"],
-					"rxInfo_longitude_0": value["rxInfo_longitude_0"],
-					"rxInfo_rssi_0": value["rxInfo_rssi_0"],
-					"txInfo_dataRate_spreadFactor": value["txInfo_dataRate_spreadFactor"],
-					"txInfo_frequency": value["txInfo_frequency"],
-					"data_humidity": value["data_humidity"],
-					"data_temperature": value["data_temperature"],
+				"data_counter_0d_0":            value["data_counter_0d_0"],
```

Comment:
Please let all "data_VALUES" in sequence, alfabetic order.

Reply:
Changed as requested